images/kor/README.md

```diff
@@ -0,0 +1,65 @@
+<!--monopod:start-->
+# kor
+| | |
+| - | - |
+| **OCI Reference** | `cgr.dev/chainguard/kor` |
+
+
+* [View Image in Chainguard Academy](https://edu.chainguard.dev/chainguard/chainguard-images/reference/kor/overview/)
+* [View Image Catalog](https://console.enforce.dev/images/catalog) for a full list of available tags.
+* [Contact Chainguard](https://www.chainguard.dev/chainguard-images) for enterprise support, SLAs, and access to older tags.*
+
+---
+<!--monopod:end-->
+
+# Kor - Kubernetes Orphaned Resources Finder
+
+Kor is a tool to discover unused Kubernetes resources. Currently, Kor can identify and list unused:
+- ConfigMaps  
+- Secrets
+- Services
+- ServiceAccounts
+- Deployments
+- StatefulSets
+- Roles
+- HPAs
+- PVCs
+- Ingresses
+- PDBs
+- CRDs
+
+![Kor Screenshot](/images/screenshot.png)
```

Comment:
This seems to be a broken link. We can just remove this though, we don't normally include screenshots in these readmes.